auth-service/pom.xml

```diff
@@ -33,12 +33,16 @@
     <dependencies>
         <dependency>
             <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-starter-web</artifactId>
+            <artifactId>spring-boot-starter-webflux</artifactId>
```

Comment:
Using webflux won't be a good idea for the auth service. Let's revert back to Spring Web MVC.